docs/articles/documentation/extending-testcafe/custom-reporter/index.md

```diff
@@ -38,21 +38,20 @@ mkdir my-reporter
 cd my-reporter
 ```
 
-It is recommended to name the directory the same as you would like to name your reporter project. When you run the generator,
-it will automatically suggest the reporter name that matches the reporter directory name. As for the reporter package name,
-it will consist of two parts - the `{testcafe-reporter-}` prefix and the name of the reporter itself:
-
-```text
-{testcafe-reporter-my-reporter}
-```
+> It is recommended to name the directory the same as you would like to name your reporter project. When you run the generator,
+it will automatically suggest the reporter name that matches the reporter directory name.
+>
+> The generator will also automatically create the reporter package name that consists of two parts - the `testcafe-reporter-` prefix and the name of the reporter itself, for example,`testcafe-reporter-my-reporter`.
+>
+> **Important:** If you name the reporter package manually, it is required that its name starts with the `testcafe-reporter-` prefix. Otherwise, TestCafe will be unable to recognize the plugin.
 
 Then run the reporter generator to create a new project:
 
 ```bash
 yo testcafe-reporter
 ```
 
-The generator will provide additional settings to customize your reporter.
+The generator will ask you for information about the reporter.
```

Comment:
no-no, I didn't mean literally `asks for information`, sorry... I must've been unclear...